codemods/convertMDXImages/README.md

```diff
@@ -0,0 +1,114 @@
+# Image Codemod
+
+## Introduction
+
+### Context
+
+The english version of the site contains subdirectors of images at each leaf of a docs directory. For example, `src/content/docs/apm` may contain an `images/` subdirectory that, when translated, are duplicated into `src/i18n/content/{locale}/docs/apm`. Not only may the images become duplicated, the same exact image may be referenced in multiple different content directories. This can lead to one single file being multiplied `X` amount of times referenced by `Y` amount of times duplicated. The purpose of this codemod is to alias all images inside `.mdx` files to an absolute path that lives at `src/images`, where all english and translated content can be referenced to this one source of truth.
```

Comment:
```suggestion
The english version of the site contains subdirectories of images at each leaf of a docs directory. For example, `src/content/docs/apm` may contain an `images/` subdirectory that, when translated, are duplicated into `src/i18n/content/{locale}/docs/apm`. Not only may the images become duplicated, the same exact image may be referenced in multiple different content directories. This can lead to one single file being multiplied `X` amount of times referenced by `Y` amount of times duplicated. The purpose of this codemod is to alias all images inside `.mdx` files to an absolute path that lives at `src/images`, where all english and translated content can be referenced to this one source of truth.
```
```suggestion
The english version of the site contains subdirectors of images at each leaf of a docs directory. For example, `src/content/docs/apm` may contain an `images/` subdirectory that, when translated, are duplicated into `src/i18n/content/{locale}/docs/apm`. Not only may the images become duplicated, the same exact image may be referenced in multiple different content directories. This can lead to one single file being multiplied `X` amount of times referenced by `Y` amount of times duplicated. The purpose of this codemod is to alias all images inside `.mdx` files to an absolute path that lives at `src/images`, where all english and translated content can be referenced to this one source of truth.
```

Reply:
idk why thats double 🙈 

but a question I had: does `src/content/docs/apm` count as a leaf?

Reply:
It does not - it should definitely be more explicit

Reply:
i was also thinking you could just say something more general: the site has duplicate images, duplicate image subdirectories, etc.